### DISCUSSION TITLE
Logged out after a phone restart

### DISCUSSION THREAD

**Discussion by EDM115** [community/community#12422] [Mobile] [answered]
When we have restarted our phone (intentionally or after a crash), whatever if we were using the GitHub app or not, it will log us out.  
It will simply prompts the log in screen, like if we just have installed the app.  
  
Fortunately, logging in back is simple (one click and it's done)

**Comment by Iron-Ham** [comment, marked as answer]
Hi @EDM115,

Can you provide some details about the device? Is this the Android or iOS app, and have you seen this only on one device or multiple?

**Reply by EDM115** [reply]
Android app, only on my device for now  
`Android Oreo MR1(27) armeabi-v7a`  
Will check on my friend's phones ☺️

**Reply by Iron-Ham** [reply]
Thank you for that information -- could you also let me know what version of the Android app you're using? 

I'm going to open up an internal issue to track this shortly!

**Reply by EDM115** [reply]
v.1.53.0

**Reply by EDM115** [reply]
edit : also exist on latest version `1.73.0-beta (10077)`

**Reply by EDM115** [reply]
`v1.77.0`  
seems fixed
also I changed my phone to another one in Android 10, maybe that's why
